ontopy/excelparser.py

```diff
@@ -187,179 +221,20 @@ def create_ontology_from_pandas(  # pylint:disable=too-many-locals,too-many-bran
-                            raise ExcelError(
-                                f'Invalid parents for "{name}": {exc}\n'
-                                "Have you forgotten an imported ontology?"
-                            ) from exc
-                        invalid_parent = True
-                        break
-                    else:
-                        parents.append(parent)
-
-                if invalid_parent:
-                    continue
-
-                if not parents:
-                    parents = [owlready2.Thing]
-
-                try:
-                    concept = onto.new_entity(name, parents)
-                except LabelDefinitionError:
-                    concepts_with_errors["wrongly_defined"].append(name)
-                    continue
-
-                added_rows.add(index)
-                # Add elucidation
-                try:
-                    _add_literal(
-                        row,
-                        concept.elucidation,
-                        "Elucidation",
-                        only_one=True,
-                    )
-                except AttributeError as err:
-                    if force:
-                        _add_literal(
-                            row,
-                            concept.comment,
-                            "Elucidation",
-                            only_one=True,
-                        )
-                        warnings.warn("Elucidation added as comment.")
-                    else:
-                        raise ExcelError(
-                            f"Not able to add elucidations. {err}."
-                        ) from err
-
-                # Add examples
-                try:
-                    _add_literal(
-                        row, concept.example, "Examples", expected=False
-                    )
-                except AttributeError:
-                    if force:
-                        warnings.warn(
-                            "Not able to add examples. "
-                            "Did you forget to import an ontology?."
-                        )
-
-                # Add comments
-                _add_literal(row, concept.comment, "Comments", expected=False)
-
-                # Add altLabels
-                try:
-                    _add_literal(
-                        row, concept.altLabel, "altLabel", expected=False
-                    )
-                except AttributeError as err:
-                    if force is True:
-                        _add_literal(
-                            row,
-                            concept.label,
-                            "altLabel",
-                            expected=False,
-                        )
-                        warnings.warn("altLabel added as rdfs.label.")
-                    else:
-                        raise ExcelError(
-                            f"Not able to add altLabels. " f"{err}."
-                        ) from err
+    # Add concepts
```

Comment:
Just a minor comment: In OWL are concepts called classes, so I would have called it `_add_class()`